dynamic_desirability/desirability_uploader.py

```diff
@@ -2,13 +2,15 @@
 import argparse
 import json
 import os
+import time
 import shutil
 import subprocess
 import bittensor as bt
 from typing import List, Optional
 from decimal import Decimal, ROUND_HALF_UP
+from substrateinterface.exceptions import SubstrateRequestException
```

Comment:
what is this? and why do we need this?

Reply:
If the priority is too low for the transaction to go through in the pool (SubstrateRequestException: Priority is too low...) then it tries again. I'll change to general exception so that we don't need to import this.